### PR TITLE
WINTER to demo data

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds WINTER to demo data.

### CODE REVIEW

File: data/db_demo.yaml [hunk 1]
```diff
@@ -491,6 +499,18 @@ instrument:
...
     # field_data:
     #   file: ../data/GROWTH-India_Fields.csv.bz2
+  - band: optical
+    name: WINTER
+    telescope_id: =WINTER-1m
+    type: imager
+    =id: WINTER
+    api_classname: GENERICAPI
+    api_classname_obsplan: MMAAPI
+    filters: ["desy", "2massj", "2massh"]
```
Comment (robertdstein) [inline data/db_demo.yaml:509]: What does "desy" mean here?

**Review by robertdstein — COMMENTED** [review]
I am assuming you do not gave a different IR category?

If not, then it all looks fine, though the "filters:desy" confused me a little.

**Comment by mcoughlin** [comment]
@robertdstein You get to choose from this list basically: https://sncosmo.readthedocs.io/en/stable/bandpass-list.html. Presumably the measured bandpasses can go into sncosmo down the line.

File: data/db_demo.yaml (hunk 1 above)
Comment (mcoughlin) [reply data/db_demo.yaml:509]: desy is the dark energy survey y, we just need to choose the closest for now.

Comment (robertdstein) [reply data/db_demo.yaml:509]: Sure, that sounds reasonable!

**Review by robertdstein — APPROVED** [review]

**Pull request merged**